core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -75,10 +72,10 @@ export class ExplorationHtmlFormatterService {
 
   getAnswerHtml(
       answer: string, interactionId: string,
-      interactionCustomizationArgs: InteractionArgs): string {
+      interactionCustomizationArgs: IInteractionCustomizationArgs): string {
     // TODO(sll): Get rid of this special case for multiple choice.
     var interactionChoices = null;
-    if (interactionCustomizationArgs.choices) {
+    if ('choices' in interactionCustomizationArgs) {
```

Comment:
Please use `.hasOwnProperty` instead, here and elsewhere.

Reply:
There are typescript errors if I use that. Btw, any specific reason to use that. Using `in` the typescript infers automatically the possible cases where `choices` may be present.

Reply:
Hmm OK, as long as we stay consistent this is fine with me then.